coffee_lib/src/plugin.rs

```diff
@@ -165,6 +165,19 @@ impl Plugin {
     pub fn tipping_info(&self) -> Option<Tipping> {
         self.conf.as_ref().and_then(|conf| conf.tipping.clone())
     }
+
+    pub fn important(&self) -> bool {
+        let config = if let Some(conf) = &self.conf {
+            conf
+        } else {
+            return false;
+        };
+        if let Some(important) = config.plugin.important {
+            important
+        } else {
+            false
+        }
```

Comment:
```suggestion
        let config = if let Some(conf) = &self.conf {
            if let Some(important) = config.plugin.important {
                important
             } else {
                 false
             }
        } else {
            return false;
        };
```

We can still do better but at least this is cleaner